addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -221,7 +221,9 @@ function factory(dependencies) {
             this.update({
                 thread: link(thread),
             });
-            this.focus();
+            if (!this.env.device.isMobileDevice) {
+                this.focus();
+            }
```

Comment:
Discuss component is not used to display threads in mobile, this should be in chat window instead no?

I guess it doesn't hurt to have it here too, but it likely doesn't resolve the issue.

Reply:
I added the check inside chat window, feel you free to merge when it's ok for you :+1: